src/storage/txn/commands/flush.rs

```diff
@@ -123,7 +123,8 @@ impl Flush {
             // txn_size is unknown, set it to max to avoid unexpected resolve_lock_lite
             txn_size: u64::MAX,
             lock_ttl: self.lock_ttl,
-            min_commit_ts: TimeStamp::zero(),
+            // min_commit_ts == 0 will disallow readers pushing it
+            min_commit_ts: self.start_ts.next(),
```

Comment:
Should the `min_commit_ts` be passed from the `Flush` kv request?

Reply:
It could be. But in current implementation is we don't have to. We can change it in the future when needed.

Reply:
It's better to use the `min_commit_ts` from the `FlushRequest` I think, otherwise that field would always be coverd.